mypy_django_plugin/transformers/querysets.py

```diff
@@ -58,7 +58,9 @@ def get_field_type_from_lookup(
     except LookupsAreUnsupported:
         return AnyType(TypeOfAny.explicit)
 
-    if (isinstance(lookup_field, RelatedField) and lookup_field.column == lookup) or isinstance(
+    if lookup_field is None:
+        return AnyType(TypeOfAny.implementation_artifact)
```

Comment:
IMO `implementation_artifact` is slightly off, but I'm not finding any other choice a better option here really.